src/lib/openapi/spec/feature-events-schema.ts

```diff
@@ -0,0 +1,28 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { eventSchema } from './event-schema';
+import { tagSchema } from './tag-schema';
+
+export const featureEventsSchema = {
+    $id: '#/components/schemas/featureEventsSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['toggleName', 'events'],
+    properties: {
+        version: { type: 'number' },
```

Comment:
The `version` field isn't specified by the TS type. I'm not sure whether it's intended to be served or if it's a mistake.